src/core/doe_commercial_reference_building_loads.jl

```diff
@@ -296,3 +296,51 @@ function custom_normalized_flatload(doe_reference_name, year)
     return normalized_profile
 end
 
+"""
+    get_monthly_energy(; power_profile::AbstractArray{<:Real,1}=Real[],
+                        normalized_profile::AbstractArray{<:Real,1}=Real[],
+                        annual_energy::Float64=nothing,
+                        year::Int64=2017)
+Get monthly energy from an hourly load profile.
+"""
+function get_monthly_energy(; power_profile::AbstractArray{<:Real,1}=Real[],
+                            normalized_profile::AbstractArray{<:Real,1}=Real[],
+                            annual_energy::Float64=nothing,
+                            year::Int64=2017)
+    t0 = 1
+    monthly_fraction = zeros(12)
+    if isnothing(annual_energy)
+        annual_energy = sum(power_profile)
+    elseif !isempty(power_profile)
+        annual_energy = sum(power_profile)
+        @warn "Using sum(power_profile) for annual_energy instead of input annual_energy"
+    end
```

Comment:
I'm taking a stab at simplifying. I was thinking it could be used for different applications with annual_energy and to get monthly_fractions, but I'm going to remove all that and just use it for the immediate need.